src/main/java/seedu/address/model/food/Carbohydrate.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.model.food;
+
+public class Carbohydrate extends Macronutrient {
+
+    public Carbohydrate(int amount) {
+        super(amount, 4);
```

Comment:
I think we should turn the `4` into a constant, i.e. `private static final CALORIC_MULTIPLIER = 4`. Same for the rest of the macronutrient classes